### DISCUSSION TITLE
How can I install 3rd party packages

### DISCUSSION THREAD

**Discussion by HUSTlwr** [exodrifter/unity-python#26] [Q&A] [answered]
I want to import "xlrd" in my python code, but I don't how to install it to my Unity project. Is there someway install 3rd party packages. Thank you.

![2bbc9359e475accb1eb0573087093cc](https://user-images.githubusercontent.com/50614735/79136629-e61e1200-7de3-11ea-8636-f6bff7214637.png)

**Comment by exodrifter** [comment, marked as answer]
You'll need to add the location of the `xlrd` module when you set up the Python engine. We do something like this in the [PythonLibrary.cs example](https://github.com/exodrifter/unity-python/blob/0.4.1/Examples/PythonLibrary.cs).

Basically, you'll need to do something like this:
```cs
var engine = Python.CreateEngine();

var paths = engine.GetSearchPaths();
paths.Add("path/to/xlrd/module");
engine.SetSearchPaths (paths);
```

**Comment by HUSTlwr** [comment]
OK, It worked. Thanks
